downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -230,6 +230,20 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 - Introduced new APIs for RSA-KEM-ML-KEM-768 hybrid public-key encryption (HPKE).
 - Introduced new APIs for ML-DSA65 (Dilithium3) signing.
 
+#### `edi` package
+
+- Added support for field length constraints (min/max) to enhance schema validation capabilities.
+- Introduced support for an escape character in EDI, allowing more flexibility in data formatting and transmission.
+
+#### `edi` Tool package
```

Comment:
What's the difference between EDI and EDI tool packages? And if this `edi Tool package` is not a ballerina/ballerinax library module, wouldn't it be a better option to have it under the developer tooling section?